diracx-core/src/diracx/core/config/schema.py

```diff
@@ -159,6 +159,56 @@ class OperationsConfig(BaseModel):
     ResourceStatus: dict[str, Any] | None = None
 
 
+class DBConfig(BaseModel):
+    DBName: str
+    Host: str
+    Port: int
+
+
+class JobDBConfig(DBConfig):
+    MaxRescheduling: int = 3
+
+
+class DatabasesConfig(BaseModel):
+    JobDB: JobDBConfig
+    JobLoggingDB: DBConfig
+    PilotAgentsDB: DBConfig
+    SandboxMetadataDB: DBConfig
+    TaskQueueDB: DBConfig
+    ElasticJobParametersDB: DBConfig
+    VirtualMachineDB: DBConfig
+
+
+class ProductionConfig(BaseModel):
+    URLs: dict[str, Any] | None = None
+    Services: dict[str, Any] | None = None
+    Agents: dict[str, Any] | None = None
+    JobWrapper: dict[str, Any] | None = None
+    Databases: DatabasesConfig
+    Executors: dict[str, Any] | None = None
+    FailoverURLs: dict[str, Any] | None = None
+
+
+class WorkloadManagementConfig(BaseModel):
+    Production: ProductionConfig | None = None
```

Comment:
I think you can delete this part (since https://github.com/DIRACGrid/DIRAC/pull/7186  was merged)
```suggestion
class WorkloadManagementConfig(BaseModel):
    Production: ProductionConfig | None = None
```